jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/TearDownStep.java

```diff
@@ -1,18 +1,120 @@
 package com.cerner.jwala.ui.selenium.steps;
 
+import com.cerner.jwala.common.exec.RemoteSystemConnection;
+import com.cerner.jwala.common.jsch.JschService;
 import com.cerner.jwala.ui.selenium.SeleniumTestHelper;
 import cucumber.api.java.After;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 
 import java.io.IOException;
-import java.sql.SQLException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
 
 /**
  * Created by Jedd Cuison on 8/7/2017
  */
 public class TearDownStep {
 
+    @Autowired
+    private JschService jschService;
+
+    @Autowired
+    @Qualifier("seleniumTestProperties")
+    private Properties props;
+
     @After
     public void afterScenario() throws SQLException, IOException, ClassNotFoundException {
+        final List<ServerInfo> webServerNameList;
+        final List<ServerInfo> jvmNameList;
+
+        try {
+            webServerNameList = getStoppedWebServers();
+            jvmNameList = getStoppedJvms();
+        } catch (final IOException | ClassNotFoundException | SQLException e) {
+            throw new TearDownException("There was an error in retrieving the list of web servers from the database!", e);
+        }
+
+        final String sshUser = props.getProperty("ssh.user.name");
+        final String sshPwd = props.getProperty("ssh.user.pwd");
+
+        System.setProperty("PROPERTIES_ROOT_PATH", this.getClass().getResource("/selenium/vars.properties").getPath()
+                .replace("/vars.properties", ""));
+        for (final ServerInfo serverInfo : webServerNameList) {
+            final RemoteSystemConnection remoteSystemConnection
+                    = new RemoteSystemConnection(sshUser, sshPwd, serverInfo.host, 22);
+            jschService.runShellCommand(remoteSystemConnection, "sc delete " + serverInfo.name, 300000);
+        }
```

Comment:
For now no since the generate deploy that Rahul wrote that I refactored stops the service. If you are going to have a scenario that only does the start, then we should add the stopping of the services in the cleanup.